documentation/source/documentation/cheatsheets/python.rst

```diff
@@ -0,0 +1,600 @@
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    map(proc, str1, str2)         |    map(proc, str1, str2)                      |
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    for a,b in zip(list1, list2): |    do(proc, list1, list2)                     |
+|      proc(a, b)                  |                                               |
++----------------------------------+-----------------------------------------------+
+
+Continuations
+-------------
+
+Python doesn’t have first-class continuations like Scheme’s
+``call/cc``, and Dylan’s ``block`` syntax is more like structured
+exception handling or generators.
+
+Here are approximate equivalents:
+
++-----------------------------------+-----------------------------------------------+
+| Python                            | Dylan                                         |
++===================================+===============================================+
+| .. code-block:: python            | .. code-block:: dylan                         |
+|                                   |                                               |
+|    def example():                 |    block (k)                                  |
+|      try:                         |      body                                     |
+|        body()                     |    exception (e :: <error>)                   |
+|      except Error:                |      handle-error()                           |
+|        handle_error()             |    cleanup                                    |
+|      finally:                     |      cleanup-stuff                            |
+|        cleanup_stuff()            |    end block                                  |
++-----------------------------------+-----------------------------------------------+
+| N/A                               | .. code-block:: dylan                         |
+|                                   |                                               |
+|                                   |    define function top ()                     |
+|                                   |      block (exit-block)                       |
+|                                   |        bar(exit-block);                       |
+|                                   |        format-out("You won't see this.\n");   |
+|                                   |      end;                                     |
+|                                   |      format-out("Top done\n");                |
```

Comment:
In this context I guess it would be clearer to say `format-out("You WILL see this.\n")`